agent/src/agent/cli/prompt/pipeline/directory.py

```diff
@@ -9,6 +9,7 @@ def prompt_config(self):
             raise click.UsageError('No api key configured. Please configure api key using `agent destination` command')
         super().prompt_config()
         self.prompt_flush_bucket_size()
+        self.pipeline.protocol = self.pipeline.destination.PROTOCOL_30
```

Comment:
is it set when you create the pipeline with a file?

Reply:
yes, it's set when we load json config into a pipeline